### PR TITLE
Mock PyCondorAPI

### PR DESCRIPTION
**Pull request by amaltaro**
Well, if I got all this mocking stuff right, then it should mock the condor apis :-)
@ticoann please review. I'll need it for #8250

### CODE REVIEW

File: src/python/WMQuality/Emulators/EmulatedUnitTestCase.py [hunk 1]
```diff
@@ -94,4 +96,13 @@ def setUp(self):
             self.inUseMemoryCache = self.memoryCachePatcher.start()
             self.addCleanup(self.memoryCachePatcher.stop)
 
+        if self.mockPyCondor:
+            self.condorPatcher = mock.patch('WMComponent.JobSubmitter.JobSubmitterPoller.ReqMgrAux', new=MockPyCondorAPI)
```
Comment (ticoann) [inline src/python/WMQuality/Emulators/EmulatedUnitTestCase.py]: WMComponent.JobSubmitter.JobSubmitterPoller.ReqMgrAux

Shouldn't this be

WMCore.Services.PyCondor.PyCondorAPI

Comment (amaltaro) [reply src/python/WMQuality/Emulators/EmulatedUnitTestCase.py]: Yes :) Thanks!

**Comment by amaltaro** [comment]
test this please

**Comment by amaltaro** [comment]
test this please

**Comment by amaltaro** [comment]
test this please

**Pull request merged**